plugins/services/src/js/components/ServiceConfigEnvironmentVariablesSectionDisplay.js

```diff
@@ -68,7 +78,7 @@ class ServiceConfigEnvironmentVariablesSectionDisplay extends ServiceConfigBaseS
             return (
               <Table
                 key="secrets-table"
-                className="table table-simple table-break-word flush-bottom"
+                className="table table-simple table-align-top table-break-word table-fixed-layout flush-bottom"
```

Comment:
Ah I see. Ok I agree then, just make sure there is an even spacing from above and below the label in single-line rows 👍